bin/tessel-install-drivers.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var logs = require('../lib/logs');
+var child_process = require('child_process');
+
```

Comment:
@rwaldron but I want to keep the check within the `install` function. Otherwise, as soon as `bin/tessel-2.js` imports this file, the CLI will exit.